spec/service_consumers/pact_helper.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'pact/provider/rspec'
+
+require 'service_consumers/provider_states'
+
+# Ensure your provider application version enables you to trace back to an exact
+# state of your provider codebase.
+# The easiest way to do this is to include the build number (or a SHA) in your version.
+provider_version = ENV['GIT_COMMIT'] || `git rev-parse --verify HEAD`
+# publish_flag = ENV['PUBLISH_VERIFICATION_RESULTS'] == "true"
+
+Pact.service_provider 'VA.gov API' do
+  # This example points to a local file, however, on a real project with a continuous
+  # integration box, you would use a [Pact Broker](https://github.com/pact-foundation/pact_broker) or publish your pacts as artifacts,
+  # and point the pact_uri to the pact published by the last successful build.
+  # honours_pact_with 'VA.gov' do
+  #   pact_uri 'tmp/va.gov-va.gov_api.json'
+  # end
+
+  app_version provider_version
+  publish_verification_results true
```

Comment:
The broker needs a version for both the consumer and provider to properly tabulate the results. Provider verson can be anything we want, but commit hashes seemed like a solid way of setting the version for the broker. I'm still not sure of the exact deploy strategy we want to enforce, though, in terms of when we can tell a consumer that it is good to deploy based on the version of the provider that passed verification.

The flag to turn on the publishing of verification results can be tied to deploy environment if we wanted.